dashboard/test/ui/features/teacher_tools/rubrics/teacher_view_of_rubric.feature

```diff
@@ -51,36 +53,69 @@ Scenario: Teachers can give and send feedback on the rubric to students.
   And I wait until element "textarea:contains(Nice work Lillian!)" is visible
 
-  @eyes
-  Scenario: Teacher views Rubric and Settings tabs
-    Given I create a teacher-associated student named "Aiden"
-    And I sign in as "Teacher_Aiden" and go home
-    And I add the current user to the "ai-rubrics" single section experiment for the "allthethings" course
-    And I wait until element "#homepage-container" is visible
-    And element "#sign_in_or_user" contains text "Teacher_Aiden"
-    And I add the current user to the "ai-rubrics" single user experiment
-    And I am on "http://studio.code.org/s/allthethings/lessons/48/levels/2"
-    And I wait for the page to fully load
-    And element ".teacher-panel td:eq(1)" contains text "Aiden"
-    And I click selector ".teacher-panel td:eq(1)" to load a new page
-    And I wait for the page to fully load
+Scenario: Teacher views rubric product tour
+  # Teacher signs in and views product tour
+  Given I create a teacher-associated student named "Aiden"
+  And I sign in as "Teacher_Aiden" and go home
+  And I add the current user to the "ai-rubrics" single section experiment for the "allthethings" course
+  And I wait until element "#homepage-container" is visible
+  And element "#sign_in_or_user" contains text "Teacher_Aiden"
+  And I add the current user to the "ai-rubrics" single user experiment
+  And I am on "http://studio.code.org/s/allthethings/lessons/48/levels/2"
+  And I wait for the page to fully load
+  And element ".teacher-panel td:eq(1)" contains text "Aiden"
+  And I click selector ".teacher-panel td:eq(1)" to load a new page
+  And I wait for the page to fully load
+  And I wait until element "h1:contains(Getting Started with AI Teaching Assistant)" is visible
+  And I wait until element ".introjs-tooltiptext" is visible
+  And I click selector ".introjs-button:contains(Next Tip)" once I see it
+  And I wait until element "h1:contains(Understanding the AI Assessment)" is visible
+  And I wait until element ".introjs-tooltiptext" is visible
+  And I click selector ".introjs-button:contains(Next Tip)" once I see it
+  And I wait until element "h1:contains(Using Evidence)" is visible
+  And I wait until element ".introjs-tooltiptext" is visible
+  And I click selector ".introjs-button:contains(Next Tip)" once I see it
+  And I wait until element "h1:contains(Understanding AI Confidence)" is visible
+  And I wait until element ".introjs-tooltiptext" is visible
+  And I click selector ".introjs-button:contains(Next Tip)" once I see it
+  And I wait until element "h1:contains(Assigning a Rubric Score)" is visible
+  And I wait until element ".introjs-tooltiptext" is visible
+  And I click selector ".introjs-button:contains(Next Tip)" once I see it
+  And I wait until element "h1:contains(How did we do?)" is visible
+  And I wait until element ".introjs-tooltiptext" is visible
+  And I click selector ".introjs-button:contains(Done)" once I see it
```

Comment:
to complete the scenario, I would suggest additional steps which verifies that we're now back into a mode where the teacher can interact with the rubric UI. For example:
* verify contents of rubrics UI matches what it should look like for the current level (as opposed to the tour)
* switch tabs or learning goals, and confirm that the UI updates correspondingly

since this has been tested manually, this seems fine to do as a fast-follow.

Reply:
Ah, the high z-index is actually part of the introjs module. I had to override their built-in css for some of our styling, so some elements (like the ones with the crazy z-index) are copied straight from their css.